src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1797,6 +1799,32 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
 
 
 
+    // =============================
+    // === Node Level Navigation ===
+    // =============================
+
+    frp::extend! { network
+
+        target_to_enter <- inputs.enter_hovered_node.map(f_!(scene.mouse.target.get()));
+
+        // Go level up pn background click.
+        enter_on_background <- target_to_enter.map(f!([](target)
+            if *target == display::scene::Target::Background { Some(()) } else { None }
+        )).unwrap();
```

Comment:
Write this instead (simpler and more FRP-style):
```rust
enter_on_bg <= tgt_to_enter.map(|tgt| *tgt == (display::scene::Target::Background).as_some()))
```

I think we could simplify it even more to:

```rust
enter_on_bg <= tgt_to_enter.map(|tgt| tgt.is_background().as_some())
```